bridge-token-factory/src/lib.rs

```diff
@@ -196,13 +255,34 @@ impl BridgeTokenFactory {
                 >= required_deposit + self.bridge_token_storage_deposit_required
         );
 
-        ext_bridge_token::mint(
-            new_owner_id,
-            amount.into(),
-            &self.get_bridge_token_account_id(token),
-            env::attached_deposit() - required_deposit,
-            MINT_GAS,
-        )
+        match parse_recipient(new_owner_id) {
+            Recipient::AccountId(account_id) => ext_bridge_token::mint(
+                account_id,
+                amount.into(),
+                &self.get_bridge_token_account_id(token),
+                env::attached_deposit() - required_deposit,
+                MINT_GAS,
+            ),
+            Recipient::ContractId {
+                evm_address,
+                message,
+            } => ext_bridge_token::mint(
+                env::current_account_id(),
+                amount.into(),
+                &self.get_bridge_token_account_id(token),
+                env::attached_deposit() - required_deposit,
```

Comment:
Should we send the deposit to `mint` or to `ft_transfer_call`? The destination account may need to be created by `ft_transfer_call`.

Reply:
ft_transfer_call must receive exactly 1 yoctoNEAR. So target account in Aurora must exist before initiating this transaction.